service/src/main/resources/ttf/Montserrat-Bold.ttf.license

```diff
@@ -0,0 +1,93 @@
+Copyright 2011 The Montserrat Project Authors (https://github.com/JulietaUla/Montserrat)
```

Comment:
What you actually for these files, is like:

```txt
SPDX-FileCopyrightText: 2011 The Montserrat Project Authors (https://github.com/JulietaUla/Montserrat)

SPDX-License-Identifier: OFL-1.1
```

and let the tool download these actual license files under `LICENSES/`. Looks like currently the tool doesn't really care about the contents of `*.license` files but that's what they should be (see existing files).

Reply:
Ah Thanks!